.travis.yml

```diff
@@ -11,8 +11,7 @@ install:
 - sudo apt-get install jq
 - cd ./bin && npm install && cd ../
 script:
-- "./bin/deploy-categories.sh"
-- "./bin/deploy-articles.sh"
+- "echo \"test\""
```

Comment:
What is this for?

Reply:
So that every file doesn't get redeployed. If the original scripts were left in then it would deploy every file that got moved and the build will time out.

Reply:
Yikes!